kernel/constr.ml

```diff
@@ -895,9 +925,11 @@ let compare_head_gen_leq_with kind1 kind2 leq_universes leq_sorts eq leq nargs t
     && Array.equal_norefl (eq 0) tl1 tl2 && Array.equal_norefl (eq 0) bl1 bl2
   | CoFix(ln1,(_,tl1,bl1)), CoFix(ln2,(_,tl2,bl2)) ->
     Int.equal ln1 ln2 && Array.equal_norefl (eq 0) tl1 tl2 && Array.equal_norefl (eq 0) bl1 bl2
+  | Array(_u1,t1,def1,ty1), Array(_u2,t2,def2,ty2) -> (* FIXME compare u1 and u2 *)
```

Comment:
@SkySkimmer would you mind taking care of this one?

Reply:
done